### PR TITLE
Fix : update split pattern for search term highlighted text [CPCN-18]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: assets/wire/utils.js [hunk 1]
```diff
@@ -205,7 +205,7 @@ export function shortHighlightedtext(html, maxLength = 40) {
   
     while (node && count < maxLength) {
         const content = node.textContent.trim();
-        const words = content.split(/\s/).filter(w => w);
```
Comment (MarkLark86) [inline assets/wire/utils.js:208]: Why change from split by `whitespace` to split by `followed by [A-Z]`?

Comment (devketanpro) [reply assets/wire/utils.js:208]: actually that split not give us desired output in the case if search term is found in the last line of the paragraph and paragraph is too long.

Comment (MarkLark86) [reply assets/wire/utils.js:208]: It would be good to have some unit tests for this

Comment (devketanpro) [reply assets/wire/utils.js:208]: sure

**Review by MarkLark86 — APPROVED** [review]

**Pull request merged**